backend/app/services/rag/pipeline/pandoc.py

```
@@ -0,0 +1,288 @@
+    """
+
+    # File extensions handled by this pipeline
+    SUPPORTED_EXTENSIONS = {".doc", ".docx", ".ppt", ".pptx"}
+
+    # Pandoc input format mapping
+    INPUT_FORMAT_MAP = {
+        ".doc": "doc",
+        ".docx": "docx",
+        ".ppt": "pptx",  # Pandoc uses pptx for both
+        ".pptx": "pptx",
+    }
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)